app/models/publisher.rb

```diff
@@ -44,7 +44,7 @@ class Publisher < ApplicationRecord
 
   attr_encrypted :authentication_token, key: :encryption_key
 
-  validates :email, email: { strict_mode: true }, presence: true, unless: -> { pending_email.present? || deleted? }
+  validates :email, email: true, presence: true, unless: -> { pending_email.present? || deleted? }
```

Comment:
We either need to remove this validation or update the existing one because we have this in the publisher.rb file already.
```ruby
  validates :email,
    format: { with: URI::MailTo::EMAIL_REGEXP },
    presence: true,
    allow_nil: true,
    uniqueness: { case_sensitive: false },
    unless: -> { deleted? }
```
Could be that there was a merge conflict that made this reappear 🤔

Reply:
If you search `EmailValidator` in https://guides.rubyonrails.org/active_record_validations.html this is what they recommend.

Reply:
We can go for the recommended approach then :)